README.md

```diff
@@ -1,3 +1,18 @@
 # happraisal
 
-Happraisal is an application that makes people working on the appraisal **h**appy.
\ No newline at end of file
+Happraisal is an application that makes people working on the appraisal **h**appy.
+
+
+# List of improvements
+
+- [] Add Unit tests (Junit5, Mockito, AssertJ)
+- [] Add Integration tests  (MockMvc, TestContainer)
+- [] Add Component test (Rest Assured)
+- [] Add sonar configuration
+- [] Validation of code quality (spotless / checkstyle / google-java-format)
```

Comment:
google-java-format isn't for validating, instead it formats the code.
Could you please add a new item with "Code formatting"?